incubator-tools/cmek_docai_processor/cmek_docai_processor.ipynb

```diff
@@ -0,0 +1,909 @@
+{
```

Comment:
<p>Is there a reason we're making all of these samples for Key functions?</p><p> </p><p>They are all available in the public documentation here</p><p> </p><p><a href="https://cloud.google.com/kms/docs/create-key-ring#kms-create-key-ring-python" target="_blank">https://cloud.google.com/kms/docs/create-key-ring#kms-create-key-ring-python</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/749/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes , most of the functions are available in public documentation but for  a flow how to use CMEK key with  DocAI is clearly explained in this (we got few requests from customer for this , so we thought it would be better if this flow is available in GIT)